src/App.js

```diff
@@ -0,0 +1,34 @@
+import {
+  BrowserRouter, Routes, Route, Link,
+} from 'react-router-dom';
+import './App.css';
+import BookList from './components/BookList';
+import BookForm from './components/BookForm';
+
+function App() {
+  return (
+    <>
+
+      <BrowserRouter>
+        <h1>Awesome books</h1>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Add of books | </Link>
+            </li>
+            <li>
+              <Link to="/categories"> Categories  </Link>
+            </li>
+          </ul>
+        </nav>
+        <Routes>
+          <Route path="/" element={<BookForm />} />
+          <Route path="/categories" element={<BookList />} />
```

Comment:
The app.js component is suppose to contain the 2 page components: <Books> and <Categories>. However, I see that the category component is missing. Kindly upate to reflect this requirement. :wink: